packages/devextreme/js/__internal/grids/grid_core/ai_column/ai_prompt_editor/ai_prompt_editor.ts

```
@@ -202,16 +208,17 @@ export class AiPromptEditor {
    * @param action - The current action being performed
    */
   public updateStateOnAction(
-    action?: AiPromptEditorAction,
+    action: AiPromptEditorAction,
   ): void {
+    // eslint-disable-next-line default-case
```

Minor: Usually by default we can throw new exception "Unknown command"